src/App.jsx

```diff
@@ -1,35 +1,26 @@
 import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+// import reactLogo from './assets/react.svg'
+// import viteLogo from '/vite.svg'
+// import './App.css'
+import LoginPage from './components/login/LoginPage.tsx'
+import ResetPassword from './components/reset-password/ResetPassword.tsx'
+import ConfirmPassword from './components/reset-password/ConfirmPassword.tsx'
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [currentPage, setCurrentPage] = useState('reset'); // 'reset' or 'confirm'
```

Comment:
Is you are using typescript we can make sure the type is known.


eg: `useState<"reset" | "confirm">`